examples/tests/valid/website_with_api.w

```diff
@@ -0,0 +1,63 @@
+bring cloud;
+
+//needs to be written before the website (so the website will be able to use it's url on sim env)
+let api = new cloud.Api();
+
+let website = new cloud.Website( path: "./website_with_api" );
+
+let users_table = new cloud.Table(cloud.TableProps{
+    name: "users-table",
+    primary_key: "id",
+    columns: {
+      id: cloud.ColumnType.STRING,
+      name: cloud.ColumnType.STRING,
+      age: cloud.ColumnType.NUMBER,
+    }
+  });
+
+
+let get_handler = inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
+    return cloud.ApiResponse {
+      body: {users: users_table.list()},
+      status: 200
+    };
+  };
+
+let post_handler = inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
+    let body: Json = req.body ?? {name: "", age: "", id: ""};
+    if (body.get("name") == "" || body.get("age")  == "" || body.get("id")  == "") {
+        return cloud.ApiResponse {
+            body: {error: "incomplete details"},       
+            status: 500
```

Comment:
nit: http code for bad user input is 400 (bad request). 
In general 400-s are for user error, 500-s are for server error
```suggestion
            status: 400
```